packages/document-routes/entrypoint.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-
+set -e 
```

Comment:
We usually use the following as best practice
```suggestion
set -eou pipefail 
```